### ISSUE TITLE
[Rating History Graph] - control is not visible in page

### ISSUE THREAD

**Issue by onetliviunicolae** [appirio-tech/tc-site#436]
Environment

-----------------------------------------------------------------------

Operating System(s): Windows 7
Device(s): Laptop HT 635
Browser(s): Chrome 49.0.2623.112 m
                 Mozilla Firefox 48.0.2  
         

-----------------------------------------------------------------------

Description

- Access application link https://members.topcoder.com and login with a valid user.

- Access application link https://www.topcoder.com and login with a valid user.

- Select Dashboard.

- Press on Content Creation contest(my example) you can use any else example.

- Observe the Rating History Graph.

----------------------------------------------------------------------

Outcome

- the control is not visible and can not be used. 

Expected Outcome

- all controls must be available.

-----------------------------------------------------------------------

see attach!

![1w](https://cloud.githubusercontent.com/assets/5114907/18395532/774ced64-76c7-11e6-9344-07ad86304e0b.png)

![2w](https://cloud.githubusercontent.com/assets/5114907/18395572/a2f68a6a-76c7-11e6-8c54-7cda09c40cdf.png)

![3ws](https://cloud.githubusercontent.com/assets/5114907/18395579/a950bad4-76c7-11e6-86ee-23c021ec14e7.png)

**Comment by tladendo** [comment]
didn't see a difference w/ "www"

**Comment by onetliviunicolae** [comment]
Correct. No difference between the same browsers. The issue can be simulated with Mozilla Firefox and Chrome.
